website/content/docs/commands/pki/health-check.mdx

```diff
@@ -234,7 +259,13 @@ This health check checks each role to see whether or not it allows wildcard issu
 
  - `allowed_roles` `(list: nil)` - an allow-list of roles to ignore.
 
-This health check checks each role to see whether it sets `no_store=false` or not. When used with lots of certificates and no temporal auto-rebuilding of CRLs, this can result in bad performance. Instead, BYOC can be used for the occasional revocation and shorter certificate lifetimes used instead. Without auto-rebuild, this will be a warning, but when auto-rebuild is enabled, we can make it informational.
+This health check checks each role to see whether it sets `no_store=false` or not. When used with lots of certificates and no temporal auto-rebuilding of CRLs, this can result in bad performance. Instead, [BYOC](/vault/api-docs/secret/pki#revoke-certificate) can be used for the occasional revocation and shorter certificate lifetimes used instead. Without auto-rebuild, this will be a warning, but when auto-rebuild is enabled, we can make it informational.
```

Comment:
```suggestion
Checks each role to see whether `no_store` is set to `false`.

<Important>
Vault will provide warnings and performance will suffer if you have a large
number of certificates without temporal CRL auto-rebuilding and set `no_store`
to `true`.
</Important>
```

The other information is semi-duplicated in the remediation section so I merged and simplified it.